src/it/scala/com/exasol/cloudetl/kafka/docker/KafkaImportIT.scala

```diff
@@ -0,0 +1,161 @@
+package com.exasol.cloudetl.kafka
+
+import java.sql.ResultSet
+import java.util.UUID
+
+import com.exasol.dbbuilder.dialects.Table
+import com.exasol.matcher.ResultSetStructureMatcher.table
+import com.exasol.matcher.TypeMatchMode.NO_JAVA_TYPE_CHECK
+
+import org.hamcrest.Matcher
+import org.hamcrest.MatcherAssert.assertThat
+import org.scalatest.BeforeAndAfterEach
+
+class KafkaImportIT extends BaseKafkaDockerIntegrationTest with BeforeAndAfterEach {
+
+  private[this] val schemaName = "KAFKA_SCHEMA"
+  private[this] val tableName = "KAFKA_TABLE"
+  private[this] var topicName: String = _
+
+  private[this] def getTableName(): String = s""""$schemaName"."$tableName""""
+
+  override final def beforeAll(): Unit = {
+    super.beforeAll()
+    prepareExasolDatabase(schemaName)
+    updateExasolHostsFile()
+  }
+
+  override final def beforeEach(): Unit = {
+    topicName = "kafka-topic-" + UUID.randomUUID()
+    createTopic(topicName)
+  }
+
+  override final def afterEach(): Unit = {
+    executeStmt(s"DROP TABLE IF EXISTS ${getTableName()}")
+    deleteTopic(topicName)
+  }
+
+  test("import string values") {
+    KafkaImportChecker(Map("COLUMN" -> "VARCHAR(20)"))
+      .withTopicValues(Seq("a", "b", "c"))
+      .assert(
+        table()
+          .row("a", 0L, 0L)
+          .row("b", 0L, 1L)
+          .row("c", 0L, 2L)
+          .matches(NO_JAVA_TYPE_CHECK)
+      )
+  }
+
+  test("import with CONSUME_ALL_OFFSET parameter empty topic") {
+    KafkaImportChecker(Map("COLUMN" -> "VARCHAR(20)"))
+      .withImportProperties(Map("CONSUME_ALL_OFFSETS" -> "true"))
+      .assert(table("VARCHAR", "BIGINT", "DECIMAL").matches())
+  }
+
+  test("import with CONSUME_ALL_OFFSET parameter") {
+    val properties = Map(
+      "CONSUME_ALL_OFFSETS" -> "true",
+      "MAX_POLL_RECORDS" -> "2",
+      "MIN_RECORDS_PER_RUN" -> "1",
+      "MAX_RECORDS_PER_RUN" -> "2"
+    )
+    KafkaImportChecker(Map("COLUMN" -> "VARCHAR(20)"))
+      .withImportProperties(properties)
+      .withTopicValues(Seq("one", "two", "three"))
+      .assert(
+        table()
+          .row("one", 0L, 0L)
+          .row("two", 0L, 1L)
+          .row("three", 0L, 2L)
+          .matches(NO_JAVA_TYPE_CHECK)
+      )
+  }
+
+  def updateExasolHostsFile(): Unit = {
```

Comment:
Should this also go to Base class?

Reply:
I was not sure about this method yet, I think I am going to refactor it in #9.